src/mapSearch/mapSearchComponent.tsx

```diff
@@ -287,9 +317,23 @@ const MapSearchComponent = ({
 
                 return (
                   <Fragment key={subtype.id}>
-                    <h3 className="MapSearchComponent__plot-search-subtype-heading">
-                      {subtype.name}
-                    </h3>
+                    <div className="MapSearchComponent__plot-search-subtype-heading">
```

Comment:
Something to talk with the designer about, I guess: this looks kinda wonky with some of the longer search type titles. There simply isn't enough room for both the title and the guide.

<img width="573" alt="image" src="https://github.com/City-of-Helsinki/mvj-ui-public/assets/12729676/e99ce844-d2de-47db-b466-7318ab8dc337">